### PR TITLE
[10.0][IMP] website_portal_medical_prescription_order_line: Upgrade view

### PR DESCRIPTION
**Pull request by kellylougheed**
- Changes prescription table in website_portal_medical_prescription_order_line into responsive prescription cards.

prescription-cards

### CODE REVIEW

File: website_medical_prescription_order_line/views/medical_prescription_order_line_template.xml [hunk 1]
```diff
@@ -19,83 +18,60 @@
...
+            <div class="container">
+                <t t-foreach="prescription_order_lines" t-as="line">
+                    <div class="col-xs-12 col-sm-6">
+                        <div class="panel panel-default">
+                            <div class="panel-heading">
+                                <span t-field="line.patient_id.name" /> - <span t-field="line.medicament_id.display_name" />
+                            </div>
+                            <div class="panel-body">
+                                <label class="label label-info">Rx Number</label>
+                                <span t-field="line.name" /><br/>
```
Comment (lasley) [inline website_medical_prescription_order_line/views/medical_prescription_order_line_template.xml]: General rule - if you're using a `<br/>`, you're probably doing it wrong. Similar to tables, really. There's a time & a place, but for the most part we design around them.

In this case, I believe encapsulating the label and spans in `<p>` would a slightly better spacing too.
```
<p>
    <label />
    <span />
</p>
```

File: website_medical_prescription_order_line/views/medical_prescription_order_line_template.xml [hunk 2]
```diff
@@ -19,83 +18,60 @@
...
+
+                                <label class="label label-info">Remain</label>
+                                <span t-field="line.can_dispense_qty" /><br/>
+
                                 <t t-if="line.last_dispense_id.date_planned">
-                                   <span t-field="line.last_dispense_id.date_planned" t-field-options='{"widget": "date"}' />
+                                    <label class="label label-info">Last Disp</label><br/>
+                                    <span t-field="line.last_dispense_id.date_planned" t-field-options='{"widget": "date"}' />
```
Comment (lasley) [inline website_medical_prescription_order_line/views/medical_prescription_order_line_template.xml]: I don't think we want the date widget here -It's not an editable field. Using it also makes it a block element, so it's on a sep line:

image

File: website_medical_prescription_order_line/views/medical_prescription_order_line_template.xml [hunk 3]
```diff
@@ -19,83 +18,60 @@
...
-                                               class="product_id"
-                                               name="product_id"
-                                               t-att-value="int(product.product_variant_ids[0]) if len(product.product_variant_ids) == 1 else '0'"
-                                               />
+                                                class="product_id"
+                                                name="product_id"
+                                                t-att-value="int(product.product_variant_ids[0]) if len(product.product_variant_ids) == 1 else '0'"
+                                                />
                                         <input type="hidden" name="set_qty" t-att-value="line.qty" />
                                         <t t-if="line.id in order_lines_filtered.ids or [line.id] == order_lines_filtered.ids">
```
Comment (lasley) [inline website_medical_prescription_order_line/views/medical_prescription_order_line_template.xml]: Please add a note to readme:
* Refactor around requirement for pre-existing Sale Order for Add to Cart button

**Review by lasley — CHANGES_REQUESTED** [review]
Thanks @kellylougheed - comments inline.

I think we may also want to switch up to two columns inside the KanBan in order to save some vertical space.

**Comment by kellylougheed** [comment]
Changes made in response to the inline comments @lasley!

File: website_portal_medical_prescription_order_line/views/medical_prescription_order_line_template.xml [hunk 4]
```diff
@@ -6,7 +6,6 @@
 -->
 
 <odoo>
-
```
Comment (woodbrettm) [inline website_portal_medical_prescription_order_line/views/medical_prescription_order_line_template.xml]: Should be an empty line between the odoo tags and the content inside them.
```
<odoo>

    <template, record, etc>

</odoo>
```

**Review by woodbrettm — APPROVED** [review]
LGTM! Checked on my phone as well and looks nice.

File: website_portal_medical_prescription_order_line/views/medical_prescription_order_line_template.xml (hunk 4 above)
Comment (kellylougheed) [reply website_portal_medical_prescription_order_line/views/medical_prescription_order_line_template.xml]: Empty lines added!

**Review by lasley — APPROVED** [review]

**Pull request merged**